### DISCUSSION TITLE
Suspicious GitHub clone

### DISCUSSION THREAD

**Discussion by kitswas** [community/community#134208] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

The following website is an exact imitation of GitHub
https://git.homegu.com/
Since it has a login page that imitates GitHub, I assume the purpose is to grab user credentials.
The website is based in China.

I could not find a way to inform GitHub, so **I am posting this here to warn people**.

**Comment by queenofcorgis** [comment]
Hi there - thanks for your vigilance and reporting, please open a ticket on our [Support page](https://support.github.com/contact).

**Reply by kitswas** [reply]
I posted this here **_because_** I landed here from the link you provided.

None of the categories on that page cover this case.

**Reply by queenofcorgis** [reply, marked as answer]
@kitswas thanks for clarifying, I reached out for further guidance. Our recommendation since this is outside of GitHub's domain is to report the page as phishing so they can evaluate the site. As a reminder, be diligent and always exercise caution where you're entering your GitHub credentials.

**Reply by kitswas** [reply]
I reported it to Google following https://github.com/orgs/community/discussions/40302#discussioncomment-4257495

**Reply by cegranger** [reply]
Same thing here with https://github.toolshello.com. I reported it to name.com their registrar. What is weird is that it was the first result from a google search for github "name_of_repository".
